test/spec/protractor_sync_test.ts

```diff
@@ -510,6 +512,19 @@ describe('Protractor extensions', () => {
 
       expect(el.hasClass('second')).toEqual(true);
     }));
+
+    it('re-selects the 2nd visible element', createTest(() => {
+        const el = elementSync.findVisible('.stale-test-3');
+
+        browserSync.executeScript(() => {
+            const stale = document.querySelector('.stale-test');
+            stale.parentNode.removeChild(stale);
+        });
+
+        appendStaleTestArea('second');
+
+        expect(el.hasClass('second')).toEqual(true);
+    }));
```

Comment:
So I may be overthinking this one.  I see what the test is doing here and it looks good!  
( Re-adding the `.second` class and verifying that the element contains it. )

What I'm not sure about is when I add this `protractor_sync_test.ts` file back to my local develop, I'm expecting the test to fail (pre `selection.ts` change above) but it's succeeding.

I've cleaned the build and re-ran the build.  Not sure if I'm missing something.

Thanks!

EDIT:  Never mind the above, I didn't have the test spec file in the right spot.  I was able to recreate the test error earlier.  Approving.